scico/solver.py

```diff
@@ -1,11 +1,60 @@
 # -*- coding: utf-8 -*-
-# Copyright (C) 2020-2021 by SCICO Developers
+# Copyright (C) 2020-2022 by SCICO Developers
 # All rights reserved. BSD 3-clause License.
 # This file is part of the SCICO package. Details of the copyright and
 # user license can be found in the 'LICENSE' file distributed with the
 # package.
 
-"""Optimization algorithms."""
+"""Optimization algorithms.
```

Comment:
Suggestion: "Scipy optimization algorithms" or similar to differentiate between this and `scico.optimize` which currently has the same description.

Reply:
Very good point. Done.